### PR TITLE
Clean up inclusion of IE-specific JS files.

### PR DESCRIPTION
**Pull request by XhmikosR**
Move them to footer, and remove duplicate ones.

/CC @cvrebert

### CODE REVIEW

File: docs/examples/tooltip-viewport/index.html [hunk 1]
```diff
@@ -45,8 +45,6 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
     <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>
     <script src="../../dist/js/bootstrap.min.js"></script>
-    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
```
Comment (cvrebert) [inline docs/examples/tooltip-viewport/index.html]: @XhmikosR The header copies should be removed, not the footer ones.

Comment (XhmikosR) [inline docs/examples/tooltip-viewport/index.html]: Indeed, I messed it up with the rebases. I'll fix it now.

**Comment by XhmikosR** [comment]
Updated.

**Comment by cvrebert** [comment]
:shipit:

**Pull request merged**